virtual-desktop/src/app/window-manager/simple-window-manager/simple-window-manager.service.ts

```diff
@@ -24,6 +25,7 @@ export class SimpleWindowManagerService implements MVDWindowManagement.WindowMan
   readonly windowResized: EventEmitter<{ width: number, height: number }>;
   private applicationManager: MVDHosting.ApplicationManagerInterface;
   private pluginDef: MVDHosting.DesktopPluginDefinition;
+  private authenticationManager: any;
```

Comment:
It is possible that by altering the type info here that it could break plugins that currently use it.
I don't want to touch it at this time, please. I know this wont cause a runtime issue but it could cause a built time issue.